extensions/skins/conversation_v1/player.html

```diff
@@ -202,6 +202,21 @@ <h4 class="conversation-skin-responses-dropdown-container" ng-click="toggleShowP
         margin-left: 0;
       }
     }
+    @-webkit-keyframes oppia-animate-bounce-dot  {
+      0% { -webkit-transform: scale(1); }
+      50% { -webkit-transform: scale(0.9); }
+      0% { -webkit-transform: scale(1); }
+    }
+    @-moz-keyframes oppia-animate-bounce-dot  {
```

Comment:
[This page](http://www.w3schools.com/cssref/css3_pr_animation-keyframes.asp) seems to suggest only -webkit is needed.